end_to_end/3-mitigate-bias-train-model2-registry-e2e.ipynb

```diff
@@ -0,0 +1,1340 @@
+{
```

Comment:
<p>Again, introduce the Bias Report as also introduced in this notebook: https://github.com/aws/amazon-sagemaker-examples/blob/master/sagemaker_processing/fairness_and_explainability/fairness_and_explainability.ipynb</p><p> </p><p>This section only shows the top 5 features, not the full report.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
<p>correct, we only will consider the top 5. And correct I have now referenced the above notebook. thx</p><p> </p>

Reply:
<p>correct.that is only how much we would like to show for practical reasons</p>